Exercise_8/index.js

```diff
@@ -0,0 +1,24 @@
+import contentService from './src/services/contentService';
+import historyService from './src/services/historyService';
```

Comment:
You can put `index.js` to the root of `./src/services` folder and export both services there. Then the import will look like this:
`import { contentService, historyService } from './src/services';`